### DISCUSSION TITLE
Plugins vs. Web Implementation

### DISCUSSION THREAD

**Discussion by dardanbujupaj** [ionic-team/capacitor#4320] [Q&A] [answered]
A few of the capacitor plugins (e.g. Toast, ActionSheet, Dialog) provide Components which are also available with a lot of UI Frameworks (Ionic has ActionSheet, Toast and Modal; Angular Material has BottomSheet, Snackbar and Dialog).

Are there reasons beyond the native look to use the capacitor plugins of the web solutions?

**Comment by jcesarmobile** [comment, marked as answer]
Capacitor uses native views when possible, and not all all people use frameworks that have those web equivalents.
But if your framework provides them and you are happy with them, they you can use those instead of the native ones.

**Reply by dardanbujupaj** [reply]
Thanks for the reply!
Will the usage of the native views bring noticeable better performance in those cases?

**Reply by jcesarmobile** [reply]
I have not tried any web UI nor done any performance comparison, so I can't say.
